### PR TITLE
package: make adapter an ES6 module

### PR DESCRIPTION
**Pull request by fippo**
which can be imported as
```
import adapter from 'webrtc-adapter';
```

### CODE REVIEW

**Comment by fippo** [comment]
@dagingaa can you give this a try? Works for me that way. I also wonder if we should get rid of the dist directory, I am not very fond of committing artifacts -- which are on the gh-pages branch anyway.

**Comment by fippo** [comment]
joy... so the ES6 import works but now browserify doesn't work anymore... :-)

File: Gruntfile.js [hunk 1]
```diff
@@ -31,12 +31,12 @@ module.exports = function(grunt) {
       // Use this if you do not want adapter to expose anything to the global
       // scope.
       adapterAndNoGlobalObject: {
```
Comment (fippo) [inline Gruntfile.js:33]: pondering dropping this...

**Comment by fippo** [comment]
@legastero said it works for him a week back or so

**Comment by dagingaa** [comment]
Works for me, I tried it and found out I had to change my webpack config to resolve from "module".

Old config:
`mainFields: ["browser", "jsnext:main", "main"],`

New config:
`mainFields: ["browser", "module", "main"],`

**Comment by fippo** [comment]
fab, will release soon.

**Comment by fippo** [comment]
note: publish like this: https://docs.npmjs.com/adding-dist-tags-to-packages

**Pull request merged**